de.fu_berlin.inf.dpp.server/src/de/fu_berlin/inf/dpp/server/ServerLifecycle.java

```diff
@@ -74,4 +78,16 @@ private void connectToXMPPServer(final ContainerContext context) {
     ConnectionHandler connectionHandler = context.getComponent(ConnectionHandler.class);
     connectionHandler.connect(account, false);
   }
+
+  public void setConsole(final ServerConsole console) {
+    this.console = console;
+  }
+
+  public void initConsole(final ContainerContext context) {
+    if (console == null) {
+      return;
+    }
+
+    console.registerCommand(new InviteCommand(context.getComponent(ISarosSessionManager.class)));
```

Comment:
Why dont you add the console to the context/container ? This is why we use DI/IOC to prevent those boilerplate code.

Reply:
I will take a look at it. I just quickly hacked something together after your suggestion of a `ContextLifecycle` broke my initial PR.